code/game/objects/items/storage/pouch.dm

```diff
@@ -1052,7 +1052,10 @@
 		to_chat(usr, SPAN_WARNING("There is no container inside this pouch!"))
 		return
 
-	usr.put_in_any_hand_if_possible(inner, disable_warning = TRUE)
+	var/had_empty_hand = usr.put_in_any_hand_if_possible(inner, disable_warning = TRUE)
```

Comment:
Use /mob/proc/put_in_hands(obj/item/W, drop_on_fail = TRUE)

Should handle dropping if there's no free hand.

Reply:
That is the first thing I tried but it doesn't work because that function resets the item's plane to its initial value which for this object is not the dropped item plane, see video:


https://github.com/cmss13-devs/cmss13/assets/59719612/b3363f98-bca6-4cfc-ad8e-90374f489d51

Reply:
Huh, right you are. You could probably put the proc in the if but this is fine.